src/main/kotlin/no/nav/familie/ef/iverksett/brev/JournalførVedtaksbrevTask.kt

```diff
@@ -85,30 +98,38 @@ class JournalførVedtaksbrevTask(private val iverksettingRepository: Iverksettin
                         iverksett.vedtak.beslutterId
                 ).journalpostId
 
-                tilstandRepository.oppdaterJournalpostResultatBrevmottakere(
+                tilstandRepository.oppdaterJournalpostResultat(
```

Comment:
```    

private fun journalførOgLagreResultat(behandlingId, arkiverDokumentRequest, personIdent, beslutterId) {
                val journalpostId = journalpostClient.arkiverDokument(arkiverDokumentRequest, beslutterId)
                tilstandRepository.oppdaterJournalpostResultat(
                        behandlingId = behandlingId,
                        mottakerIdent = personIdent,
                        JournalpostResultat(journalpostId = journalpostId)
                )
}
private fun journalførVedtaksbrevTilBrevmottakere(iverksett: Iverksett,
                                                      journalførteIdenter: List<String>,
                                                      arkiverDokumentRequest: ArkiverDokumentRequest,
                                                      behandlingId: UUID) {
        iverksett.vedtak.brevmottakere?.mottakere?.mapIndexed { indeks, mottaker ->
            if (!journalførteIdenter.contains(mottaker.ident)) {
                        val arkiverDokumentRequestForMottaker = arkiverDokumentRequest.copy(
                                eksternReferanseId = "$behandlingId-vedtaksbrev-mottaker$indeks",
                                avsenderMottaker = AvsenderMottaker(
                                        id = mottaker.ident,
                                        idType = mottaker.identType.tilIdType(),
                                        navn = mottaker.navn
                                )
                        )
                        journalførOgLagreResultat(behandlingId, arkiverDokumentRequestForMottaker, mottaker.ident, iverksett.vedtak.beslutterId)
                        }

private fun journalførVedtaksbrevTilStønadmottaker(arkiverDokumentRequest: ArkiverDokumentRequest,
                                                       iverksett: Iverksett,
                                                       behandlingId: UUID) {
                                                       
        journalførOgLagreResultat(behandlingId, arkiverDokumentRequest, iverksett.søker.personIdent, iverksett.vedtak.beslutterId)
    }
```

Reply:
Kronglete, men tror det blir lettere å lese forskjellen på de to

Reply:
Bra forslag! Refaktorerer.